### PR TITLE
Basic metrics dashboard

### PR DESCRIPTION
**Pull request by arfon**
This PR adds a collection of basic metrics views show high level stats for papers going through the JOSS system. 

screen shot 2018-08-18 at 7 45 27 am
screen shot 2018-08-18 at 7 46 07 am
screen shot 2018-08-18 at 7 46 17 am

### CODE REVIEW

File: app/models/paper.rb [hunk 1]
```diff
@@ -23,11 +25,11 @@ class Paper < ActiveRecord::Base
     end
 
     event :start_meta_review do
-      transitions :from => :submitted, :to => :review_pending, :after => :create_meta_review_issue
+      transitions :from => :submitted, :to => :review_pending, :if => :create_meta_review_issue
```
Comment (arfon) [inline app/models/paper.rb]: This conditional makes sure that the AASM transition only happens _if_ the `create_meta_review_issue` method returns `true`

File: app/models/review_issue.rb [hunk 2]
```diff
@@ -86,8 +86,8 @@ def editor
     body.match(/\*\*Editor:\*\*\s*(@\S*|Pending)/i)[1]
   end
 
-  # Extract the Reviewer from the issue body
-  def reviewer
-    body.match(/\*\*Reviewer:\*\*\s*(@\S*|Pending)/i)[1]
+  # Extract the Reviewers from the issue body
+  def reviewers
```
Comment (arfon) [inline app/models/review_issue.rb]: This just matches the logic in [WhedonAPI](https://github.com/openjournals/whedon-api/blob/b0720d8c30673b87c5bb8860d361b403338b31e9/whedon.rb#L266-L268)

File: spec/controllers/papers_controller_spec.rb [hunk 3]
```diff
@@ -57,26 +77,53 @@
...
+      editor = create(:editor, :login => "mouse")
+      editing_user = create(:user, :editor => editor)
+
+      paper = create(:review_pending_paper, :state => "review_pending", :meta_review_issue_id => 1234, :user_id => user.id)
+      fake_issue = Object.new
+      allow(fake_issue).to receive(:number).and_return(1)
+      allow(GITHUB).to receive(:create_issue).and_return(fake_issue)
+
+      # Can't create review issue because editor is invalid
+      expect {
```
Comment (arfon) [inline spec/controllers/papers_controller_spec.rb]: Actually testing these transitions now...

File: lib/tasks/utils.rake [hunk 4]
```diff
@@ -0,0 +1,51 @@
+require "util/console_extensions"
```
Comment (arfon) [inline lib/tasks/utils.rake]: There are a couple of data migrations here.

**Comment by leeper** [comment]
Looks great!

**Comment by arfon** [comment]
OK this is now live at https://joss.theoj.org/dashboard . You can only see this if you're an editor.

**Pull request merged**